include/tango/client/device/device_command_info.hxx

```diff
@@ -0,0 +1,82 @@
+//
```

Comment:
OK,

CLion can help us with this, there is a feature to automatically add copyright. We just need to create a template and put it into the repo. So everyone can use this template in the future.